context/app/static/js/components/detailPage/DatasetRelationships/nodeTypes.tsx

```diff
@@ -66,10 +71,12 @@ function NodeTemplate({
   bgColor,
   isLoading,
   href,
+  toastText,
+  tooltipText,
 }: NodeTemplateProps) {
   // Outer wrapper Box makes sure that nodes are always the same height
   const contents = (
-    <Box height="4.125rem" display="flex" alignItems="center">
+    <Box height="4.125rem" display="flex" alignItems="center" sx={{ cursor: href ? 'pointer' : 'default' }}>
```

Comment:
We still need these styles even if we don't wrap the box in an anchor element?

Reply:
Yes - the nodes themselves get the `pointer` cursor from `react-flow` regardless of whether they're links.